app/views/root/_base.html.erb

```diff
@@ -43,15 +43,15 @@
       },
       {
         text: "Sign out",
-        href: Plek.new.website_root + "/transition-check/logout",
+        href: Plek.new.website_root + "/sign-out",
         data: {
           module: "explicit-cross-domain-links",
           link_for: "accounts-signed-in",
         },
       },
       {
         text: "Sign in",
-        href: Plek.new.website_root + "/transition-check/login",
+        href: Plek.new.website_root + "/sign-in",
```

Comment:
Add a ?redirect_path: based on the page URL?

Reply:
and also a _ga where appropraite?

Reply:
> Add a ?redirect_path: based on the page URL?

Something to check with Maxwell I think, as right now the "sign in" link takes you to the account manager.

> and also a _ga where appropraite?

The `explicit-cross-domain-links` should be doing that but... maybe it's not.

Reply:
thanks, we've nearly got the session controllers migrated.
Will get that merged then test this a few times on integration / staging.

Sould help us clarify what's needed here.

Reply:
> The `explicit-cross-domain-links` should be doing that but... maybe it's not.

I've realised why I wasn't seeing a `_ga` param on the links... I have rejected GOV.UK analytics cookies!  If I accept them, it does work.

Reply:
Have put an update on here: https://trello.com/c/43FM4pwv/673-switch-to-the-new-sign-in-out-endpoints

Will wait to hear more from design and product folks.